content/en/docs/kubernetes/operator/troubleshooting/automatic.md

```diff
@@ -0,0 +1,179 @@
+
+```yaml
+annotations:
+  instrumentation.opentelemetry.io/inject-python: 'true'
+```
+
+It tells the OpenTelemetry Operator to look for an `Instrumentation` resource in
+the pod’s namespace. It also tells the Operator to inject Python
+auto-instrumentation into the pod.
+
+When the pod starts up, the annotation tells the Operator to look for an
+`Instrumentation` resource in the pod’s namespace, and to inject Python
+auto-instrumentation into the pod. It adds an
+[init-container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
+to the application’s pod, called `opentelemetry-auto-instrumentation`, which is
+then used to injects the auto-instrumentation into the app container.
+
+But if the `Instrumentation` resource isn’t present by the time the `Deployment`
+is deployed, the `init-container` can’t be created. This means that if the
+`Deployment` resource is deployed _before_ you deploy the `Instrumentation`
+resource, the auto-instrumentation will fail to initialize.
+
+Check that the `opentelemetry-auto-instrumentation` `init-container` has started
+up correctly (or has even started up at all), by running the following command:
+
+```shell
+kubectl get events -n <your_app_namespace>
+```
+
+Which should result in output that looks something like this:
+
+```text
+53s         Normal   Created             pod/py-otel-server-7f54bf4cbc-p8wmj    Created container opentelemetry-auto-instrumentation
+53s         Normal   Started             pod/py-otel-server-7f54bf4cbc-p8wmj    Started container opentelemetry-auto-instrumentation
+```
+
+If the output is missing `Created` and/or `Started` entries for
+`opentelemetry-auto-instrumentation`, then it means that there is an issue with
+your auto-instrumentation configuration. This can be the result of any of the
+following:
+
+- The `Instrumentation` resource wasn’t installed (or wasn’t installed
+  properly).
+- The `Instrumentation` resource was installed _after_ the application was
+  deployed.
+- There’s an error in the auto-instrumentation annotation, or the annotation in
+  the wrong spot — see #4 below.
+
+You might also want to check the output of the events command for any errors, as
+these might help point to your issue.
+
+### Check the auto-instrumentation configuration
+
+You’ve added the auto-instrumentation annotation, but did you do it correctly?
+Here are a couple of things to check for:
+
+- **Are you auto-instrumenting for the right language?** For example, did you
+  try to auto-instrument a Python application by adding a JavaScript
+  auto-instrumentation annotation instead?
+- **Did you put the auto-instrumentation annotation in the right spot?** When
+  you’re defining a `Deployment` resource, there are two spots where you could
+  add annotations: `spec.metadata.annotations`, and
+  `spec.template.metadata.annotations`. The auto-instrumentation annotation
+  needs to be added to `spec.template.metadata.annotations`, otherwise _it won’t
+  work_.
```

Comment:
```suggestion
The auto-instrumentation annotation might have not been added
correctly. Check for the following:

- Are you auto-instrumenting for the right language? For example, did you
  try to auto-instrument a Python application by adding a JavaScript
  auto-instrumentation annotation instead?
- Did you put the auto-instrumentation annotation in the right location? When
  you’re defining a `Deployment` resource, there are two locations where you could
  add annotations: `spec.metadata.annotations`, and
  `spec.template.metadata.annotations`. The auto-instrumentation annotation
  needs to be added to `spec.template.metadata.annotations`, otherwise it doesn't
  work.
```